packages/sender/src/lib/sender.ts

```diff
@@ -278,6 +278,15 @@ const Sender: WalletBehaviourFactory<InjectedWallet> = async ({
           return res.response;
         });
     },
+
+    async importAccountsInSecureContext({ accounts }) {
+      if (window.near && window.near.isSender) {
+        window.near.batchImport({
```

Comment:
Based on [comments here](https://github.com/near/wallet-selector/pull/650#issuecomment-1396948993) this part need to be re-written.

Reply:
Hi, @hcho112 I think we can leave this part for now, in my opinion, what was mentioned in this comment https://github.com/near/wallet-selector/pull/650#issuecomment-1396948993 is out of the scope of this PR.